tests/conftest.py

```diff
@@ -8,48 +8,90 @@ def pytest_addoption(parser):
         if metafunc.config.option.dataset is None:
-            metafunc.parametrize("dataset", ["htqc", "cellpainting"], indirect=True)
+            metafunc.parametrize("dataset", ["htqc", "cellpainting", "qc"], indirect=True)
         else:
-            assert metafunc.config.option.dataset in ["htqc", "cellpainting"]
+            assert metafunc.config.option.dataset in ["htqc", "cellpainting", "qc"]
 
             metafunc.parametrize("dataset", [metafunc.config.option.dataset], indirect=True)
 
 
 @pytest.fixture
 def cellpainting():
     return {
+        "config": "config.ini",
         "data_dir": "tests/data_b",
-        "ingest":
+        "image_csv": "Image.csv",
+        "ingest": [
             {
-                "Image_nrows": 4,
-                "Image_ncols": 6,
-                "Cells_nrows": 40,
-                "Cells_ncols": 586,
-                "Cytoplasm_nrows": 40,
-                "Cytoplasm_ncols": 572,
-                "Nuclei_nrows": 40,
-                "Nuclei_ncols": 595
+                "ncols": 586,
+                "nrows": 40,
+                "table": "Cells"
             },
+            {
+                "ncols": 572,
+                "nrows": 40,
+                "table": "Cytoplasm"
+            },
+            {
+                "ncols": 6,
+                "nrows": 4,
+                "table": "image"
+            },
+            {
+                "ncols": 595,
+                "nrows": 40,
+                "table": "Nuclei"
+            }
+        ],
         "munge": False
     }
 
-
 @pytest.fixture
 def htqc():
     return {
+        "config": "config.ini",
         "data_dir": "tests/data_a",
         "munged_dir": "tests/data_a_munged",
-        "ingest":
+        "image_csv": "image.csv",
+        "ingest": [
+            {
+                "ncols": 294,
+                "nrows": 40,
+                "table": "Cells"
+            },
+            {
+                "ncols": 279,
+                "nrows": 40,
+                "table": "Cytoplasm"
+            },
             {
-                "image_nrows": 8,
-                "image_ncols": 229,
-                "Cells_nrows": 40,
-                "Cells_ncols": 294,
-                "Cytoplasm_nrows": 40,
-                "Cytoplasm_ncols": 279,
-                "Nuclei_nrows": 40,
-                "Nuclei_ncols": 287
+                "ncols": 229,
+                "nrows": 8,
+                "table": "image"
             },
+            {
+                "ncols": 287,
+                "nrows": 40,
+                "table": "Nuclei"
+            }
+        ],
+        "munge": True
+    }
+
+
+@pytest.fixture
+def qc():
+    return {
+        "config": None,
+        "data_dir": "tests/data_c",
+        "image_csv": "Image.csv",
+        "ingest": [
+            {
+                "nrows": 8,
+                "ncols": 229,
+                "table": "Image"
+            }
+        ],
         "munge": True
```

Comment:
Shouldn't this false `False`? @mcquin

Reply:
Either work. This is meant to test that it doesn't vomit when munge is passed (True) but there aren't any object CSVs in the config.

Reply:
Make sense. Lets leave it as is in which case